apps/web/components/eventtype/EditLocationDialog.tsx

```diff
@@ -273,20 +275,48 @@ export const EditLocationDialog = (props: ISetLocationDialog) => {
             query={locationsQuery}
             success={({ data: locationOptions }) => {
               if (!locationOptions.length) return null;
+              if (booking) {
+                locationOptions.forEach((location) => {
+                  if (location.label === "phone") {
+                    location.options.filter((l) => l.value !== "phone");
+                  }
+                });
+              }
               return (
                 <Controller
                   name="locationType"
                   control={locationFormMethods.control}
                   render={() => (
-                    <Select
+                    <Select<{ label: string; value: string; icon?: string }>
```

Comment:
i don't think this file is required anywhere. i am using editLocationDialog from @components/dialog/EditLocationDialog in event type. should i delete this ?

Reply:
Yeah doesn't look like its being used anywhere :) Feel free to delete